doc/news/_preparation_next_release.md

```diff
@@ -92,11 +92,16 @@ The text below summarizes updates to the [C (and C++)-based libraries](https://w
 - <<TODO>>
 - <<TODO>>
 
-### <<Library1>>
-
-- <<TODO>>
-- <<TODO>>
-- <<TODO>>
+### Opts
+
+- The library function `elektraGetOpts` now supports sub-commands. For example `add` is a sub-command in `git add`, and
+  interprets `-p` differently from `git`: `git -p add` is `git --paginate add`, but `git add -p` is `git add --patch`.
```

Comment:
I found it easiest to explain by using an example. I would have used `kdb`, but since it currently uses global options there is no real-world example.

Reply:
My confusion here is that I thought you are describing how `git` handles it. Only much later I found out, that your implementation and `git` handles it the same way.